src/cpp/database/database.cpp

```diff
@@ -2939,7 +2939,7 @@ DatabaseDump Database::dump_entity_(
 
```

Comment:
In this `dump` method should be added if the entity is or is not metatraffic.
Also ALIVE characteristic is not in the dump (and presumably neither in the load) and it should be.

Reply:
:+1: 
Both added to the dump, and ignored when loading.